### DISCUSSION TITLE
pcl2第一个被反馈的问题是啥啊（）

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1329] [提问] [answered]
首先，这个问题就纯粹闲的，**造成了打扰，我表示抱歉**
其次，有没有人告诉我他是啥（）

**Comment by WForst-Breeze** [comment, marked as answer]
如果你是指金数据的话，那还得让我花点翻翻看……
> 请注意：请不要在这些地方发表无关 comment！

Github **现存的***第一个 Issue 是 #38，第一个公告是 #2，第一个新功能投票是 #41，第一个 Pr 是 #85
*注：已知其实最早的 Issue 是 #3，但此 Issue 已被删除

---
（润回来补充了）
**可以考证到的**第一批被修复的反馈是 PCL#13 至 PCL#20（于 Dev 0.5.17 修复完毕）
```
· 修复了以下 7 个 Bug
    PCL-13 登录页切换动画结束前切换主页面导致动画强行停止
    PCL-14 若离线不是默认页面，则账号不会自动填入文本框
    PCL-15 初始化非当前页的设置导致单选框、滑动条显示错误
    PCL-16 非默认动画速度下加载动画错位
    PCL-17 左下角提示停留时间受动画速度影响
    PCL-18 高度可变的卡片的内容可能会消失
    PCL-20 左边栏指向颜色未能完全覆盖背景
```
没有我们想象的那样有趣，不是吗 (¬‿¬)

**Comment by iPodtouch6557** [comment]
**如果有条件的话**，麻烦翻翻金数据看？没时间发个链接也行……

**Reply by WForst-Breeze** [reply]
金数据的反馈表单处理完的反馈都会删除的（）无从考证qwq……
